app.py

```diff
@@ -0,0 +1,35 @@
+from flask import render_template, request, Flask
+import standardise
+
+app = Flask(__name__)
+
+
+@app.route("/")
+def upload():
+    return render_template("home.html")
+
+
+@app.route("/uploadSuccess", methods=["POST"])
+def uploadSuccess():
```

Comment:
I think it is ok for the endpoint name, but the function name is usually underscored according to PEP8, e.g. `upload_success`